modules/create-a-containerruntimeconfig-crd.adoc

```diff
@@ -20,15 +20,15 @@ You can modify the following settings by using a `ContainerRuntimeConfig` CR:
 * **Maximum log size**: The `logSizeMax` parameter sets the CRI-O `log_size_max` parameter, which is the maximum size allowed for the container log file. The default is unlimited (`log_size_max = -1`). If set to a positive number, it must be at least 8192 to not be smaller than  the ConMon read buffer. ConMon is a program that
 monitors communications between a container manager (such as Podman or CRI-O) and the OCI runtime (such as runc or crun) for a single container.
 
-You can create multiple `ContainerRuntimeConfig` CRs, as needed, with a limit of ten. For the first `ContainerRuntimeConfig` CR, the MCO creates a machine config appended with `containerruntime`. With each subsequent CR, the controller creates a new `containerruntime` machine config with a numeric suffix. For example, if you have a `containerruntime` machine config with a `-2` suffix, the next `containerruntime` machine config is appended with `-3`.
+You should have one `ContainerRuntimeConfig` CR for each machine config pool with all the config changes you want for that pool. You should edit an existing `ContainerRuntimeConfig` CR to modify existing settings or add new settings instead of creating a new CR for each change. It is recommended to create a new CR only to modify a different machine config pool, or for changes that are intended to be temporary, so that you can revert the changes.
```

Comment:
It's better to add this line after the first sentence:
"even if the same content is being applied to all the pools, then you only need one ContainerRuntimeConfig CR for all the pools"